packages/kbn-monaco/src/esql/lib/ast/validation/esql_validation_meta_tests.json

```diff
@@ -2319,25 +2319,33 @@
       "error": true
     },
     {
-      "query": "show",
```

Comment:
Looks like we lost this test case? Should it be translated to "meta"?

Reply:
Right, I've added the missing test with bf5f472d5c5d79fab4120eae8e75c5438c641f81